README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
```

Comment:
There should be a warning or note regarding the `task install` command, specifically its potential to run scripts that might have side effects. Users should be aware of what this command does.